test/common/ops/purchase.js

```diff
@@ -200,5 +202,24 @@ describe('shared.ops.purchase', () => {
 
       expect(user.items.gear.owned[key]).to.be.true;
     });
+
+    it.only('purchases quest bundles', () => {
+      let clock = sandbox.useFakeTimers(+moment('2017-05-20'));
```

Comment:
This ensures that we won't need to update the test when the specific bundle stops being available May 31.